src/accessor/tick.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = function() {
+  var date = function(d) { return d.date; },
+      high = function(d) { return d.high; },
+      low = function(d) { return d.low; },
```

Comment:
I though about it a lot.
High and low seem to be a describing the chart and was not stuck to the domain terminology.
I also understood the code better if I described it as high and low instead of bid-ask.